src/lib/common/utils/utils.ts

```diff
@@ -41,3 +43,29 @@ export const base64ToBigUIntString = (encoded: any) => {
     return Buffer.from(encoded, "base64").readBigUInt64BE().toString();
 };
 
+function instanceofAlgoAccount(account: any): boolean {
```

Comment:
Should this be in common/utils, or algorand/utils?

Reply:
this is currently in `common/utils/utils`